contracts/system-contracts/hedera-account-service/IHRC906Facade.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.4.9 <0.9.0;
+
+/**
+ * notice: This interface is applicable when msg.sender is an EOA or a smart contract.
+ */
+interface IHRC906Facade {
```

Comment:
I'm not sure if _Facade` will be clear to everyone. I took a bit of time to realize these methods are callable from the account directly, right?

_Facade_ make sense when you know the implementation I guess, but from a user point of view, not sure.

Reply:
Yeah `Facade` is what was noted in the HIPs so we'll have to explain it more. It's a type of proxy

Reply:
Added explanations in the README